api/utils/select_collector.go

```diff
@@ -0,0 +1,66 @@
+package utils
+
+import (
+	"github.com/DisgoOrg/disgo/api"
+	"github.com/DisgoOrg/disgo/api/events"
+)
+
+// NewSelectCollector gives you a channel to receive on and a function to close the collector
+func NewSelectCollector(disgo api.Disgo, channelID api.Snowflake, guildID api.Snowflake, messageID api.Snowflake, filter SelectFilter) (chan *api.SelectMenuInteraction, func()) {
```

Comment:
sounds better to me imo
```suggestion
func NewSelectMenuSubmitCollector(disgo api.Disgo, channelID api.Snowflake, guildID api.Snowflake, messageID api.Snowflake, filter SelectFilter) (chan *api.SelectMenuInteraction, func()) {
```